src/App.js

```diff
@@ -35,6 +35,7 @@ const App = (props) => {
         insights.chrome?.globalFilterScope?.('insights');
         if (insights.chrome?.globalFilterScope) {
             insights.chrome.on('GLOBAL_FILTER_UPDATE', ({ data }) => {
+                dispatch(setWorkloads(data?.Workloads));
                 const selectedTags = insights.chrome?.mapGlobalFilter?.(data)?.filter(item => !item.includes('Workloads')) || undefined;
                 dispatch(setSelectedTags(selectedTags));
```

Comment:
what about to join these two dispatches into one?

Reply:
OOOOO how do we do that? 🤔 🥇

Reply:
I would just create a new action, that will combine both actions (so there will be only one re-rerender)

```jsx
dispatch(globalFilterChange(data?.workloads, selectedTags))
```

Reply:
OOOOO i see `batch`